src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -789,12 +790,11 @@ DWORD InputStateMachineEngine::_GetSGRMouseModifierState(const size_t modifierPa
 // - modifierParam - the VT modifier value to convert
 // Return Value:
 // - The equivalent INPUT_RECORD modifier value.
-DWORD InputStateMachineEngine::_GetModifier(const size_t modifierParam) noexcept
+DWORD InputStateMachineEngine::_GetModifier(const VTInt modifierParam) noexcept

```

Comment:
@lhecker can you point me at the actual thing that fixes the issue here? I'm happy to merge the larger PR (I agree on the bounds check), but I'd like to know which line to point at when I feel confused :laugh: